[Elixir] Add analyzer comments for boutique-inventory (#2101)
exercism/website-copy: analyzer-comments/elixir/boutique-inventory/use_enum_map.md

# Use `Enum.map`

> One purpose of this exercise it to learn about map. Use the `Enum.map` function in the `increase_quantity` function to practice.

Reviewer: ```suggestion
Use the `Enum.map` function in the `increase_quantity` function to practice. It's the best fitting `Enum` function for this task.
```

> Use the `Enum.map` function in the `increase_quantity` function to practice. It's the best fitting `Enum` function for this task.

Reviewer 2: Yes, let's do this one in the other PR
